tox.ini

```diff
@@ -680,49 +666,23 @@ commands =
     # Checking logs generated & startup of testgateway1
     bash -c '(kubectl logs service/backend --context k3d-testgateway1 --namespace syft -f &) | grep -q "Application startup complete" || true'
 
-    # frontend
-    bash -c 'if [[ "$PYTEST_MODULES" == *"frontend"* ]]; then \
-        echo "Starting frontend"; date; \
-        pytest tests/integration -m frontend -p no:randomly -k "test_serves_domain_frontend" --co; \
-        pytest tests/integration -m frontend -vvvv -p no:randomly -p no:benchmark -o log_cli=True --capture=no -k "test_serves_domain_frontend"; \
-        return=$?; \
-        echo "Finished frontend"; date; \
-        exit $return; \
-    fi'
-
-    # Integration + Gateway Connection Tests
-    # Gateway tests are not run in kuberetes, as currently,it does not have a way to configure
-    # high/low side warning flag.
-    bash -c "source ./scripts/get_k8s_secret_ci.sh; \
-    pytest tests/integration/network -k 'not test_domain_gateway_user_code' -p no:randomly -vvvv"
-
-    # Veilid Integration tests
-    bash -c "source ./scripts/get_k8s_secret_ci.sh; \
-    pytest tests/integration/veilid -p no:randomly -vvvv"
-
-    # Shutting down the gateway cluster to free up space, as the
-    # below code does not require gateway cluster
-    bash -c "CLUSTER_NAME=testgateway1 tox -e dev.k8s.destroy || true"
-    bash -c "docker volume rm k3d-testgateway1-images --force || true"
-
-    ; container workload
-    ; bash -c 'if [[ "$PYTEST_MODULES" == *"container_workload"* ]]; then \
-    ;     echo "Starting Container Workload test"; date; \
-    ;     pytest tests/integration -m container_workload -p no:randomly --co; \
-    ;     pytest tests/integration -m container_workload -vvvv -p no:randomly -p no:benchmark -o log_cli=True --capture=no; \
-    ;     return=$?; \
-    ;     echo "Finished container workload"; date; \
-    ;     exit $return; \
-    ; fi'
```

Comment:
@rasswanth-s container workloads for k8s has been disabled. please confirm if this needs to be re-enabled.